enos/modules/vault_verify_version/scripts/verify-cluster-version.sh

```diff
@@ -31,7 +31,7 @@ if ! out=$(jq -eMc --arg version "$version" '.keys | contains([$version])' <<< "
 fi
 
 if ! out=$(jq -eMc --arg version "$version" --arg bd "$VAULT_BUILD_DATE" '.key_info[$version].build_date == $bd' <<< "$vh"); then
-  fail "cluster version history build date is not the expected date: expected: $VAULT_BUILD_DATE, output: $out"
+  fail "cluster version history build date is not the expected date: expected: true, expected date: $VAULT_BUILD_DATE, key_info: $(jq -eMc '.key_info' <<< "$vh"), output: $out"
```

Comment:
This is fairly unrelated but when I was testing a random new SHA I didn't have the build date so this failed initially. This just improves the error message to both help that development case and to also aid any future failure discovery.

Reply:
Sweet, this error message had room for improvement for sure.